include/plugin_interface/FFT_UGens.h

```diff
@@ -62,6 +62,17 @@ static inline SCComplexBuf* ToComplexApx(SndBuf* buf) {
 
 struct PV_Unit : Unit {};
 
+// Ordinary ClearUnitOutputs outputs zero, potentially telling the IFFT (+ PV UGens) to act on buffer zero, so let's
+// skip that:
+static void FFT_ClearUnitOutputs(Unit* unit, int wrongNumSamples) { ZOUT0(0) = -1; }
```

Comment:
You might use `static inline` instead of just `static`, like in the functions above.

---

As a side note:

Specifying free functions in header files as `static` or `static inline` is just a hack for C.
For C++, the more idiomatic (and efficient) specifier would be just `inline`. Note that the meaning of `inline` is different for C and C++!
For some nice explanations, see https://stackoverflow.com/a/1759575/6063908 and https://stackoverflow.com/a/68146072/6063908

Since this header is really a C++ header (it uses C++ classes), we could actually substitute all those `static inline` with `inline`, but that's not the topic of this PR :-)